core/to/util.go

```diff
@@ -0,0 +1,5 @@
+package to
+
+func Ptr[T any](v T) *T {
```

Comment:
I think there exists more than one of these:
- api/iam/api_test.go
- somewhere else?

Reply:
oh, forgot to deduplicate